Configure STK3310 as module and AXP20X_POWER built in

When the stk3311 light & proximity sensor of the pinephone is build into the kernel, the i2c communication with it fails.
This is probably because the stk3311 is powered by the PMIC (gpio0-ldo) which is not fully initialized at this time.
Index: pine64-kernel/arch/arm64/configs/pine64_defconfig
===================================================================
--- pine64-kernel.orig/arch/arm64/configs/pine64_defconfig
+++ pine64-kernel/arch/arm64/configs/pine64_defconfig
@@ -1070,7 +1070,6 @@ CONFIG_TCG_TIS_SPI=m
 CONFIG_TCG_VTPM_PROXY=m
 CONFIG_TCG_FTPM_TEE=m
 CONFIG_I2C_CHARDEV=m
-CONFIG_I2C_MUX=y
 CONFIG_I2C_MUX_GPIO=m
 CONFIG_I2C_MUX_PINCTRL=m
 CONFIG_I2C_GPIO=m
@@ -1124,7 +1123,7 @@ CONFIG_BATTERY_BQ27XXX=m
 # CONFIG_BATTERY_BQ27XXX_I2C is not set
 CONFIG_CHARGER_AXP20X=m
 CONFIG_BATTERY_AXP20X=m
-CONFIG_AXP20X_POWER=m
+CONFIG_AXP20X_POWER=y
 CONFIG_SENSORS_AD7414=m
 CONFIG_SENSORS_AD7418=m
 CONFIG_SENSORS_ADM1029=m
@@ -1342,10 +1341,10 @@ CONFIG_USB_KEENE=m
 CONFIG_USB_RAREMONO=m
 CONFIG_USB_MA901=m
 CONFIG_V4L_PLATFORM_DRIVERS=y
-CONFIG_VIDEO_SUN6I_CSI=y
+CONFIG_VIDEO_SUN6I_CSI=m
 CONFIG_V4L_MEM2MEM_DRIVERS=y
-CONFIG_VIDEO_SUN8I_DEINTERLACE=y
-CONFIG_VIDEO_SUN8I_ROTATE=y
+CONFIG_VIDEO_SUN8I_DEINTERLACE=m
+CONFIG_VIDEO_SUN8I_ROTATE=m
 CONFIG_SMS_SDIO_DRV=m
 CONFIG_VIDEO_TVAUDIO=m
 CONFIG_VIDEO_TDA7432=m
@@ -1801,7 +1800,7 @@ CONFIG_R8712U=m
 CONFIG_R8188EU=m
 CONFIG_STAGING_MEDIA=y
 CONFIG_VIDEO_SUNXI=y
-CONFIG_VIDEO_SUNXI_CEDRUS=y
+CONFIG_VIDEO_SUNXI_CEDRUS=m
 CONFIG_ASHMEM=y
 CONFIG_RTL8723CS=m
 CONFIG_COMMON_CLK_SCPI=m
@@ -1990,7 +1989,7 @@ CONFIG_OPT3001=m
 CONFIG_PA12203001=m
 CONFIG_SI1133=m
 CONFIG_SI1145=m
-CONFIG_STK3310=y
+CONFIG_STK3310=m
 CONFIG_ST_UVIS25=m
 CONFIG_TCS3414=m
 CONFIG_TCS3472=m
